packages/app/src/pages/ThemeSampleListPage.tsx

```diff
@@ -54,7 +57,13 @@ const ThemeSampleListPage = () => {
           ],
         ]}
         indexLimit={2}
-      />
+      >
```

Comment:
Kan dit niet als separate paragraaf in de ThemeSubpage? Waarom moet het als child van de tabel?

Reply:
In het design hoort de content bij het bovenstaande component. Daarnaast willen we zorgen dat styling alleen aan componenten gekoppeld is en niet lost in situaties wordt toegepast. En het is op deze manier ook makkelijk her te gebruiken.